dev-packages/application-manager/src/generator/webpack-generator.ts

```diff
@@ -39,7 +39,7 @@ const { mode }  = yargs.option('mode', {
 const development = mode === 'development';${this.ifMonaco(() => `
 
 const monacoEditorPath = development ? '${this.resolve('monaco-editor-core', 'dev/vs')}' : '${this.resolve('monaco-editor-core', 'min/vs')}';
```

Comment:
It would be good to take a look at #1542 and the referenced https://github.com/Microsoft/monaco-editor/issues/18. Basically, this means we no longer need to copy all the monaco files separately, but bundle it into our main artefact.

Reply:
yes, but it is out of the scope of this PR

Reply:
How out of scope? This is only available on Monaco 0.11+ and we should do it upon the upgrade. I'm aware that this is not a blocker, but this is not out of scope either.

Reply:
We do it with another PR, there are things to consider before switching from AMD to ESM, like the build speed. We had bad experiences in monaco-languageclient. We want consider them separately without blocking usage of the latest Monaco.